README.md

```diff
@@ -75,6 +75,7 @@ But knowing the stuff will help you become better! :muscle:*
 - :book: [Foundations of Security: What Every Programmer Needs to Know ](https://www.goodreads.com/book/show/128003.Foundations_of_Security)
 - [OWASP Top 10](https://www.owasp.org/index.php/Category:OWASP_Top_Ten_Project)
 - [Web Application Exploits and Defenses](https://google-gruyere.appspot.com/part1)
+- [A collection of hacking / penetration testing resources to make you better!](https://github.com/vitalysim/Awesome-Hacking-Resources)
```

Comment:
I don't know if hacking info is relevant in this repo. Maybe you have to add this in [Awesome Hacking Resource](https://github.com/vitalysim/Awesome-Hacking-Resources)

Reply:
It's linked to that repo which you referred to.

But I'm not sure about programmer should know some about hacking before prevent it? :sweat_smile: